config/application.rb

```diff
@@ -9,6 +9,9 @@
 
 # Require the gems listed in Gemfile, including any gems
 # you've limited to :test, :development, or :production.
+
+require 'letter_opener' if Rails.env.development?
```

Comment:
Go ahead and ditch this -- Rails knows to load everything in the Gemfile based on groups on app load, so so you're doing this twice by having letter opener in the `development` group in the actual Gemfile.